x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -113,6 +117,45 @@ export function generateMappings(fields: Field[]): Mappings {
   return { properties: props };
 }
 
+function generateMultiFields(fields: Fields): MultiFields {
+  const multiFields: MultiFields = {};
+  if (fields) {
+    fields.forEach((f: Field) => {
+      const type = f.type;
+      switch (type) {
+        case 'text':
+          multiFields[f.name] = { ...generateTextMapping(f), type: f.type };
+          break;
+        case 'keyword':
+          multiFields[f.name] = { ...generateKeywordMapping(f), type: f.type };
+          break;
+      }
+    });
+  }
+  return multiFields;
+}
+
+function generateKeywordMapping(field: Field): Mapping {
+  const mapping: Mapping = {
+    ignore_above: DEFAULT_IGNORE_ABOVE,
```

Comment:
Reminds me we should probably think about if this value is still the right one today as this is about 5 years old. @webmat I assume ECS also uses this default?

Reply:
Yes ECS still uses the default of 1024 for keyword's ignore_above.